### PR TITLE
130 b sd pi p content

### PR DESCRIPTION
**Pull request by ToddCooper**
Creating SDPi-P PR to begin final content creation work.

### CODE REVIEW

File: SDPi_Supplement/asciidoc/docinfo.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 }
 
 #toc.toc2 {
-    width: 30%;
+    width: 30em;
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/docinfo.html:10]: Is there a reason to change this to a fixed value? This breaks the responsive design I enabled for the document.

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc [hunk 2]
```diff
@@ -157,6 +159,24 @@ Each type is a source of requirements that are explicitly identified and formali
...
 
+==== Requirements Binding Strategies
+Although <<vol1_figure_appendix_a_sdpi_requirements_core_model>> generally indicates bi-directional navigation (arrows on both ends of Requirement-Usage pairs, supporting *_bi-directional bindings_* and navigation is not always helpful.
+This is especially the case when considering potential future updates to the profile specifications.
+In that case, the general rule is:
+
+[none]
+. *Add backward references from _Requirement Usage_ to _Requirement Definition_*.
+
+For example, in TF-2 Transactions, each transaction section is paired with a message transport section in #TF-2 Appendix A MDPWS#; however, future versions of the specification may provide options for alternative transports.
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc:170]: Why mark it yellow? `#TF-2 Appendix A MDPWS#`

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc [hunk 3]
```diff
@@ -157,6 +159,24 @@ Each type is a source of requirements that are explicitly identified and formali
...
+In this case, the actual transaction definition will remain unchanged, but the bindings to transport messages and services would change.
+Given the rule above, bindings are made in the current TF-2 Appendix A MDPWS profile pointing backweard (or upward!) to the transaction requirements that they satisfy.
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc:172]:
```suggestion
Given the rule above, bindings are made in the current TF-2 Appendix A MDPWS profile pointing backward (or upward) to the transaction requirements that they satisfy.
```

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc [hunk 4]
```diff
@@ -157,6 +159,24 @@ Each type is a source of requirements that are explicitly identified and formali
...
+There are no bindings in the opposite direction.
+Taking this approach, a new transport appendix could be added in the future without impacting the core transaction specifications.
+
+Application of this rule would also hold true in other places such as backward references from a profile's Use Case section to the specific #TF-1 Appendix C# use case and scenario requirements that they satisfy.
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc:176]: Yellow marking? Why not linking TF-1 Appendix C?

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-2-overview.adoc [hunk 5]
```diff
@@ -6,21 +6,57 @@ NOTE:  This file contains the Volume 1 / Chapter 2 content, including overviews
...
-// 2.x
-
-WARNING:  Need to figure out how to have a specific number vs. ".x"!
+#TODO:  DOES THIS SECTION WITH OPTIONS NEED TO BE DETAILED HERE OR IS IT SUFFICIENT TO HAVE TEMPLATE LANGUAGE POINTING TO THE http://www.ihe.net/Profiles/ PAGE?#
 
 [sdpi_offset=10]
 === Service-oriented Device Point-of-care Interoperability (SDPi) – Overview & Framework
 
-.IHE SDPi Profiles & Foundational Standards
+The Service-oriented Device Point-of-care Interoperability (SDPi) profiles are built upon a foundation of standards and profiles from <<acronym_hl7>>, <<acronym_ieee>>, IHE and other organizations.  An overview of the profiles and their relationships is provided in Figure <<figure_sdpi_profiles_foundational_standards>>:
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume1/tf1-ch-2-overview.adoc:53]:
```suggestion
The Service-oriented Device Point-of-care Interoperability (SDPi) profiles are built upon a foundation of standards and profiles from <<acronym_hl7>>, <<acronym_ieee>>, IHE and other organizations.  An overview of the profiles and their relationships is provided in Figure <<figure_sdpi_profiles_foundational_standards>>.
```

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-2-overview.adoc [hunk 6]
```diff
@@ -6,21 +6,57 @@ NOTE:  This file contains the Volume 1 / Chapter 2 content, including overviews
...
+|===
+
 // 2.2
 [sdpi_offset=2]
 === Safety, Effectiveness & Security Considerations and Requirements
+IHE specifications often include sections for "Security Considerations" and "Safety Considerations", capturing both general and specific guidance and requirements for system implementers.
+This supplement extends these two concepts to include a third:  Effectiveness.
+The sections are termed: <<term_safe_effective_secure>> Considerations.
+
+The background for <<acronym_ses>> is discussed in detail in <<vol1_appendix_a_integrating_ses>>; however, in general "<<acronym_ses>>" is used as a reference to the standards encompased (directly and indirectly by reference) in <<term_joint_working_group_7>>, including <<ref_iso_81001_1_2021>> and <<ref_iec_80001_1_2021>>.
```
Comment (d-gregorczyk) [inline SDPi_Supplement/asciidoc/volume1/tf1-ch-2-overview.adoc:38]:
```suggestion
The background for <<acronym_ses>> is discussed in detail in <<vol1_appendix_a_integrating_ses>>; however, in general "<<acronym_ses>>" is used as a reference to the standards encompassed (directly and indirectly by reference) in <<term_joint_working_group_7>>, including <<ref_iso_81001_1_2021>> and <<ref_iec_80001_1_2021>>.
```

**Review by d-gregorczyk — CHANGES_REQUESTED** [review]

File: SDPi_Supplement/asciidoc/docinfo.html (hunk 1 above)
Comment (ToddCooper) [reply SDPi_Supplement/asciidoc/docinfo.html:10]: When it was a %, it was clipping off the document's left columns.  So reverting back to a fixed width fixed the one problem but in a sub-optimal way.   I believe the plan was to do it The Right Way somewhere down the line.   For example, as I mentioned in Session #1 yesterday, eventually we would like to include all levels in an outline that collapses to the top level or two.

**Comment by ToddCooper** [comment]
@d-gregorczyk :  Note that this branch is still a work-in-progress and wasn't intended to be reviewed at this time.  Apologies for the confusion BUT thank you for the quick review!  I responded to your questions (per above).

**Comment by ToddCooper** [comment]
@d-gregorczyk :  Added a boat load of references from the use case files to the profiles BUT they generally resulted in (see [BUILD Error Log](https://github.com/IHE/sdpi-fhir/actions/runs/3745868734/jobs/6360708485#step:6:1014).

Added a list of the labels to the end of the Glossary - in case it was a forward references problem ... which didn't fix it.  

Could you give it a look at let me know what the stink went wrong?

Am now too tired to trouble shoot ... will be back in the saddle by 0500 Wednesday AFC! time.

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc (hunk 2 above)
Comment (ToddCooper) [reply SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc:205]: Updated the text with the link to Appendix A TF-2.

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc (hunk 3 above)
Comment (ToddCooper) [reply SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc:207]: Fixed

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc (hunk 4 above)
Comment (ToddCooper) [reply SDPi_Supplement/asciidoc/volume1/tf1-ch-a-requirements-interoperability.adoc:211]: Updated with the requested link.

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-2-overview.adoc (hunk 5 above)
Comment (ToddCooper) [reply SDPi_Supplement/asciidoc/volume1/tf1-ch-2-overview.adoc:61]: Changed to a period.

File: SDPi_Supplement/asciidoc/volume1/tf1-ch-2-overview.adoc (hunk 6 above)
Comment (ToddCooper) [reply SDPi_Supplement/asciidoc/volume1/tf1-ch-2-overview.adoc:36]: Corrected typo.

**Review by kenjfuchs — APPROVED** [review]

**Review by ToddCooper — COMMENTED** [review]
Reviewed & approve all changes; all requests have been addressed.

**Comment by ToddCooper** [comment]
Closed pull request due to non-availability of one reviewer to confirm requested changes were made and to Approve.

**Pull request closed without merging**